asciidoctor-stylesheet/sass/components/_asciidoc.scss

```diff
@@ -352,6 +352,9 @@ b.button:after {
   //margin-bottom: $panel-margin-bottom * 2;
   border-bottom: $sect-divider-size $sect-divider-style $sect-divider-color;
   padding-bottom: 0.5em;
+  overflow: -moz-scrollbars-none;
+  position: sticky !important; 
+  height: 570px !important; 
```

Comment:
Rather than specifying a fixed height (`570px`), please use the `calc()` function so you can calculate the height based on the user's viewport, so that it looks good on all screen sizes. 

You can calculate the height by adding up the height of the header and the footer and then subtracting the sum from `100vh` (which means 100%  of the **v**iewport **h**eight).

So, for example, if the header's height is `40px` and the footer's height is `340px` and suppose we want to keep a `20px` gap between the sidebar and the footer, then you can specify the height as `height: calc(100vh - 400px)`.

You can find the heights by inspecting them in the developer tools of the browser.